tests/e2e/specs/visit_label_printing_page.cy.js

```diff
@@ -58,15 +56,4 @@ describe('Label Printing page', () => {
     cy.get('#submit-button').click()
     cy.contains('api/label is invalid')
   })
-
-  it('PMB request is unsuccessful, empty response', () => {
-    cy.intercept('/v2/print_jobs', {
-      statusCode: 422,
-      body: {
-        errors: [],
-      },
-    })
-    cy.get('#submit-button').click()
-    cy.contains('Unknown')
-  })
 })
```

Comment:
Do you want to remove this test?

Reply:
Yes. Wasn't really clear on it's relevance. I have modified the code so that it does always return an error.